Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/FileBrowserHelper.java

```diff
@@ -85,4 +94,82 @@ public static String getSelectedPath(Intent result)
 
     return null;
   }
+
+  public static boolean isPathEmptyOrValid(StringSetting path)
+  {
+    return isPathEmptyOrValid(path.getStringGlobal());
+  }
+
+  public static boolean isPathEmptyOrValid(String path)
+  {
+    return !path.startsWith("content://") || ContentHandler.exists(path);
```

Comment:
What do you think about splitting these two checks into two separate ones that return different alert message text? Unless a user is familiar with SAF / scoped storage, they won't assume that an unavailable file is a permission issue since they already granted Dolphin the permissions it requested. I'd probably assume that a different app was locking access to the unavailable file if I didn't know about SAF / scoped storage.

Reply:
There is only one error case here: The case where the path starts with `content://` and `exists` returns false. Would it be fine if I just reworded the error message to mention permissions?

Reply:
I was thinking:

"Dolphin does not have permission to access one or more configured paths. Would you like to fix this before starting?"

Reply:
I like your suggestion.